docs/README.md

```diff
@@ -31,7 +31,7 @@ This free app keeps your balance stable and center. Lives in your Menu Bar.
 ### From the command line
 ℹ️ Requires [homebrew](https://brew.sh/) to be installed
 
-🐚 Run `brew cask install ballast`
+🐚 Run `brew install ballast`
```

Comment:
Thanks for this!

Looking at the brew formulae, https://formulae.brew.sh/cask/ballast#default, should we change it to the following? @sg-s 

```suggestion
🐚 Run `brew install --cask ballast`
```